scripts/link_bb.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+from os import path, makedirs
+from subprocess import run
+
+from qcore import simulation_structure
+from qcore import constants
+from shared_workflow.shared_automated_workflow import add_to_queue
+
+
+def arg_parser():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "source_root_dir", type=str, help="path to previous completed vaildation run"
+    )
+    parser.add_argument(
+        "destination_root_dir", type=str, help="path to the destination root directory"
+    )
+    parser.add_argument(
+        "fault_selection_list", type=str, help="a file that contains list of faults"
+    )
+
+    args = parser.parse_args()
+
+    return args
+
+
+def main(source_root_dir, destination_root_dir, fault_selection_list):
+    faults = {}
+    with open(fault_selection_list, "r") as fault_file:
```

Comment:
There's a function for this:
`from qcore.formats import load_fault_selection_file`